### DISCUSSION TITLE
Does anyone know when `DateTime modify()` returns `false`?

### DISCUSSION THREAD

**Discussion by simPod** [vimeo/psalm#8463] [Q&A]
https://github.com/vimeo/psalm/pull/8350

I did not find any example when false can happen. Does anyone know?

**Comment by AndrolGenhald** [comment]
See also #8454 and #8462, it may be worth asking this at [php-src](https://github.com/php/php-src/).

**Comment by VincentLanglet** [comment]
```
(new DateTime())->modify('potato')
```
returns false. But yeah, I'm trying to introduce a dynamic return type provider in https://github.com/vimeo/psalm/pull/8462

**Reply by simPod** [reply]
So basically when there's some gibberish. Nothing else. Right?

**Reply by VincentLanglet** [reply]
That's what we try to confirm, but yes.
